examples/tutorials/api/fitting.py

```diff
@@ -4,6 +4,17 @@
 
 Learn how the model, dataset and fit Gammapy classes work together in a detailed modeling and fitting use-case.
 
+Note
+----
```

Comment:
Could instead add a banner?
```suggestion
.. note::

Reply:
Should really put the whole Frequentist vs Bayesian discussion into a note?

Reply:
I think it's fine like this.